Makefile

```
@@ -18,14 +18,15 @@ pr: reset-anvil ##
 	$(MAKE) start-anvil-chain-with-contracts-deployed > /dev/null &
 	sleep 4 # needed to wait for anvil setup to finish
 	cargo test --workspace
-	cargo clippy --workspace --lib --examples --tests --benches --all-features
-	cargo +nightly fmt -- --check
 	docker stop anvil
 
 fireblocks-tests:
 	$(MAKE) start-anvil-chain-with-contracts-deployed > /dev/null &
 	cargo test --workspace --features fireblock-tests
 
+clippy:
+	cargo clippy --workspace --all-features  --tests --examples -- -D warnings
```

Since both `cargo fmt` and `clippy` are run together, we can include them both:
```suggestion
lint:
	cargo fmt --all -- --check \
		&& cargo clippy --workspace --all-features --benches --examples --tests -- -D warnings
```